src/shared-components/typography/index.ts

```diff
@@ -58,8 +58,14 @@ const labelStyle = (theme: ThemeType) => `
 `;
 
 const buttonStyle = (theme: ThemeType) => `
-  ${labelStyle(theme)}
-  font-weight: ${TYPOGRAPHY_CONSTANTS.fontWeight.bold};
+  color: ${theme.COLORS.primaryTint1};
+  font-size: ${TYPOGRAPHY_CONSTANTS.fontSize.button};
+  line-height: ${round(20 / 12, 2)};
```

Comment:
First three properties pulled from `labelStyle`, with the updated button font-size. 

The conditional below only adds `bold` to the primary theme.